src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -774,16 +774,10 @@ def handle_migration_parameters(command_group, server_name_arg_type, migration_i
                 elif scope == "update":
                     c.argument('migration_name', arg_type=migration_id_arg_type, options_list=['--migration-name'],
                                help='Name of the migration.')
-                    c.argument('setup_logical_replication', options_list=['--setup-replication'], action='store_true', required=False,
-                               help='Allow the migration workflow to setup logical replication on the source. Note that this command will restart the source server.')
-                    c.argument('db_names', nargs='+', options_list=['--db-names', '--dbs'], required=False,
-                               help='Space-separated list of DBs to migrate. Note that each additional DB affects the performance of the source server.')
-                    c.argument('overwrite_dbs', options_list=['--overwrite-dbs'], action='store_true', required=False,
-                               help='Allow the migration workflow to overwrite the DB on the target.')
```

Comment:
Can you just mark it as deprecated for now and then remove these parameters in our next breaking change window(which is May release for Build Event)?

Reply:
These commands are not being used for the past 3 version so its safe to remove them altogether.

Reply:
The it makes no harm to keep for another 2 versions, right? Here's our guidance for making breaking changes: https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/teams_docs/azcli_docs/breaking_changes